extensions/cornerstone-dicom-seg/src/utils/_hydrateSEG.ts

```diff
@@ -50,13 +63,15 @@ async function _hydrateSEGDisplaySet({
       segDisplaySet.displaySetInstanceUID
     );
 
+
     if (shouldDisplaySeg) {
       viewportGridService.setDisplaySetsForViewport({
         viewportIndex: index,
         displaySetInstanceUIDs: viewport.displaySetInstanceUIDs,
         viewportOptions: {
           viewportType: 'volume',
           toolGroupId,
+          presentationId: null,
```

Comment:
it doesn't belong in this PR but I guess it doesn't break anything?

Reply:
Agreed, it is in the wrong PR, but it doesn't break anything.